### PR TITLE
Fixed the links

### PR DESCRIPTION
**Pull request by Aryan-Singla**
Fixed the Links having issues like:
 1. Broken Links
 2. Redirect Links
 3. SSL Issue in Links
 4. Ignored Links

### CODE REVIEW

File: doc/ref_model/chapters/appendix-a.rst [hunk 1]
```diff
@@ -92,12 +92,12 @@ the:
...
    .org>`__.
 2. `OWASP Cheat Sheet Series (OCSS) <https://github.com/OWASP/CheatSheetSeries>`__ from the `Open Web Application
-   Security Project <https://www.owasp.org>`__.
+   Security Project <https://owasp.org/>`__.
 3. :ref:`chapters/chapter07:Workload Security - Vendor Responsibility` section of the Reference Model.
 
 The VNF Developer **should** ensure that their code is not vulnerable to the
 `OWASP Top Ten Security Risks <https://owasp.org/www-project-top-ten/>`__ created by the
-`Open Web Application Security Project <https://www.owasp.org>`__.
+`Open Web Application Security Project < https://owasp.org/>`__.
```
Comment (CsatariGergely) [inline doc/ref_model/chapters/appendix-a.rst]:
```suggestion
`Open Web Application Security Project <https://owasp.org/>`__.
```

File: doc/ref_model/chapters/chapter07.rst [hunk 2]
```diff
@@ -1105,9 +1104,8 @@ System hardening
 | req.sec.gen.002 | The systems part of the cloud infrastructure     | Hardening: CIS Password Policy Guide            |
```
Comment (CsatariGergely) [inline doc/ref_model/chapters/chapter07.rst:1104]: This should look like this: 
image

File: doc/ref_model/chapters/chapter07.rst [hunk 3]
```diff
@@ -1143,14 +1141,11 @@ System hardening
 |                 | the source code and manifests.                   |                                                 |
 +-----------------+--------------------------------------------------+-------------------------------------------------+
 | req.sec.gen.010 | The cloud infrastructure **must** support        |                                                 |
```
Comment (CsatariGergely) [inline doc/ref_model/chapters/chapter07.rst]: This should look like this: 
image

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

File: doc/ref_model/chapters/chapter07.rst [hunk 4]
```diff
@@ -171,7 +170,7 @@ important standards in current use.
   <https://www.etsi.org/technologies/nfv>`__ and its Security Working Group NFV-SEC
 
 - The International Organization for Standardization (ISO) and the International Electrotechnical Commission (IEC) -
-  `www.iso.org <https://www.iso.org>`__. The following ISO standards are of particular interest for NFVI:
+  `www.iso.org < https://www.iso.org/home.html>`__. The following ISO standards are of particular interest for NFVI:
```
Comment (CsatariGergely) [inline doc/ref_model/chapters/chapter07.rst]:
```suggestion
  `www.iso.org <https://www.iso.org/home.html>`__. The following ISO standards are of particular interest for NFVI:
```

File: doc/ref_model/chapters/chapter07.rst [hunk 5]
```diff
@@ -696,7 +695,7 @@ discovery, authentication and authorisation policies enforcement, network resili
 capabilities. The two documents,
 `NIST SP 800-204A <https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-204A.pdf>`__\ (Building Secure
 Microservices-based Applications Using Service-Mesh Architecture) and
-`NIST SP 800-204B <https://csrc.nist.gov/publications/detail/sp/800-204b/final>`__\ (Attribute-based Access Control for
+`NIST SP 800-204B < https://csrc.nist.gov/pubs/sp/800/204/b/final>`__\ (Attribute-based Access Control for
```
Comment (CsatariGergely) [inline doc/ref_model/chapters/chapter07.rst]:
```suggestion
`NIST SP 800-204B <https://csrc.nist.gov/pubs/sp/800/204/b/final>`__\ (Attribute-based Access Control for
```

File: doc/ref_model/chapters/chapter07.rst [hunk 6]
```diff
@@ -1738,7 +1733,7 @@ Compliance with standards
 |                 | Focus in Cloud Computing (latest version).   |                                                     |
 +-----------------+----------------------------------------------+-----------------------------------------------------+
 | req.sec.std.003 | The platform and workloads **should** follow | Open Web Application Security Project `https://www. |
-|                 | the guidance in the OWASP Cheat Sheet Series | owasp.org <https://www.owasp.org>`__                |
+|                 | the guidance in the OWASP Cheat Sheet Series | owasp.org < https://owasp.org/>`__                |
```
Comment (CsatariGergely) [inline doc/ref_model/chapters/chapter07.rst]:
```suggestion
|                 | the guidance in the OWASP Cheat Sheet Series | owasp.org <https://owasp.org/>`__                   |
```

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

File: doc/ref_model/chapters/chapter07.rst (hunk 3 above)
Comment (CsatariGergely) [reply doc/ref_model/chapters/chapter07.rst]: Sorry my previous suggestion was not working for the linkcheck. 

This one works:
image

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

**Comment by CsatariGergely** [comment]
@walterkozlowski I see lots of direct links in the RM Chapters which are not added via a link in the refs.bib file. In RA2 we used the practice that all links should be in the refs.bib file, so they are readeable also on paper. Shouldn't RM follow the same practice?

File: doc/ref_model/chapters/appendix-a.rst [hunk 7]
```diff
@@ -92,12 +92,12 @@ the:
 1. `CSA Security Guidance for Critical Areas of Focus in Cloud Computing (latest version) <https://cloudsecurityalliance
    .org>`__.
 2. `OWASP Cheat Sheet Series (OCSS) <https://github.com/OWASP/CheatSheetSeries>`__ from the `Open Web Application
-   Security Project <https://www.owasp.org>`__.
+   Security Project <https://owasp.org/>`__.
 3. :ref:`chapters/chapter07:Workload Security - Vendor Responsibility` section of the Reference Model.
```
Comment (CsatariGergely) [inline doc/ref_model/chapters/appendix-a.rst]:
```suggestion
3. :ref:`chapters/chapter07:Workload Security` section of the Reference Model.
```

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

File: doc/ref_model/chapters/chapter07.rst (hunk 2 above)
Comment (Aryan-Singla) [reply doc/ref_model/chapters/chapter07.rst:1104]: Solved

File: doc/ref_model/chapters/chapter07.rst (hunk 3 above)
Comment (Aryan-Singla) [reply doc/ref_model/chapters/chapter07.rst]: Solved

File: doc/ref_model/chapters/chapter07.rst [hunk 8]
```diff
@@ -1104,10 +1103,9 @@ System hardening
 +-----------------+--------------------------------------------------+-------------------------------------------------+
 | req.sec.gen.002 | The systems part of the cloud infrastructure     | Hardening: CIS Password Policy Guide            |
```
Comment (CsatariGergely) [inline doc/ref_model/chapters/chapter07.rst:1104]: This still need some fixes. I've added them as a pr to your pr: https://github.com/Aryan-Singla/RM/pull/1

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

**Review by CsatariGergely — APPROVED** [review]

**Review by petorre — APPROVED** [review]

**Review by walterkozlowski — APPROVED** [review]

**Comment by walterkozlowski** [comment]
> @walterkozlowski I see lots of direct links in the RM Chapters which are not added via a link in the refs.bib file. In RA2 we used the practice that all links should be in the refs.bib file, so they are readeable also on paper. Shouldn't RM follow the same practice?
@CsatariGergely, yes, this is how it should be.  We probably missed many.  Can we get help with this task as well?

**Pull request merged**